### ISSUE TITLE
🔥 Who reviews the papers?

### ISSUE THREAD

**Issue by ivanstepanovftw** [jiachenzhu/DyT#10]
Do you really think you can call a few trainable parameters with a tanh activation function - a "normalization"? A "replacement"? An absence?

Was there any research been done?

Are you scientists?

**Comment by ivanstepanovftw** [comment]
Even 0 prior works been cited such as "Batchless normalization".

**Comment by ivanstepanovftw** [comment]
I think this should be classified as "parametric tanh activation", not as replacement of normalization layers.

**Comment by ivanstepanovftw** [comment]
Also, after "scale and shift" there should be at least some activation. You are trying to mimic batch normalization paper with weight and bias parameters, and there was even debate "activation before of after normalization" - after normalization and always present, because weights for "scale and shift" would be neglegted by next linear layers if there is no activation between them.

Or, you could have done better ablation study and remove self.weight and self.bias. You would left with `return torch.tanh(a * input)`. A parametric hyperbolic tangent activation. Similarly like PReLU did, but you fixed 1 parameter in alpha.

Or even go further and try literally this:
```python
class DynamicTanh(nn.Module):
    def forward(self, x):
        return torch.tanh(x)
```

**Comment by ivanstepanovftw** [comment]
#2
